hr_holidays_validity_date/models/hr_holidays.py

```diff
@@ -0,0 +1,66 @@
+#     Copyright (c) 2015 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api, _
+from odoo.exceptions import ValidationError
+from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT
+from datetime import datetime
+import pytz
+
+
+class HrHolidaysStatus(models.Model):
+    _inherit = "hr.holidays.status"
+
+    date_start = fields.Datetime(string='Start Date')
+    date_end = fields.Datetime(string="Expiry Date")
+    use_validity_dates = fields.Boolean()
+    restrict_dates = fields.Boolean(string="Restrict")
```

Comment:
IMO better it is better Restrict Dates as string. It is more clear in the view.

Reply:
Sorry @jarroyomorales. This is the comment. For some reason it did not appear before. Regards.

Reply:
Changed!